Example/OriginateUI/CircleViewController.m

```diff
@@ -0,0 +1,89 @@
+//
+//  CircleViewController.m
+//  OriginateUI
+//
+//  Created by Seth Rainha on 7/7/16.
+//  Copyright © 2016 originate.com. All rights reserved.
+//
+
+#import "CircleViewController.h"
```

Comment:
If header files is in same project import in the .m file otherwise in the .h file.